bin/www/localPackage.js

```diff
@@ -140,13 +140,13 @@ var LocalPackage = (function (_super) {
                 return;
             }
             publicKey = publicKeyResult;
-            isSignatureVerificationEnabled = (publicKey !== null);
+            isSignatureVerificationEnabled = !!publicKey;
```

Comment:
You made changes in `bin` folder. This folder is generated by compiling code. Could you please make changes in source code(`www` folder) and run `gulp compile` command?

Reply:
Got it.